### PR TITLE
Sign up wording

### LINKED ISSUE #165: Change signup page text
Request: Can we change the following text for  `/signup`
change  `where do you live?` to `where should we pick up?`

### PR DESCRIPTION
**Pull request by evantbyrne**
Closes #165

### CODE REVIEW

File: lib/components/SignupCard.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ export default class SignupCard extends Component {
             <RaisedButton label="Sign Up" onClick={this.handleSubmit.bind(this)}/>
 
             <p>
-              Have an account already?
```
Comment (wdoug) [inline lib/components/SignupCard.js:51]: I don't think this is right. This is for the option to log in if the user already has an account. The change doesn't make sense.

**Comment by dviramontes** [comment]
thanks @evantbyrne !

File: lib/components/SignupCard.js (hunk 1 above)
Comment (evantbyrne) [inline lib/components/SignupCard.js:51]: You're right. That shouldn't have been changed.

**Comment by wdoug** [comment]
:shipit:

**Comment by evantbyrne** [comment]
:fireworks:

**Pull request merged**